ios/Breez Notification Service Extension/BreezManager.swift

```diff
@@ -11,55 +11,66 @@ fileprivate var log = Logger(
 fileprivate var log = Logger(OSLog.disabled)
 #endif
 
-// SDK events listener
-class SDKListener: EventListener {
+class BreezManager {
+    private static var breezSDK: BlockingBreezServices? = nil
+    fileprivate static var queue = DispatchQueue(label: "BreezManager")
+    fileprivate static var sdkListener: EventListener? = nil
     
-    var paymentListener : PaymentListener
+    static func register(listener: EventListener) throws -> BlockingBreezServices? {
+        try BreezManager.queue.sync { [] in
+            BreezManager.sdkListener = listener
+            if BreezManager.breezSDK == nil {
+                BreezManager.breezSDK = try BreezManager.connectSDK()
+            }
+            return BreezManager.breezSDK
```

Comment:
```suggestion
            return BreezManager.breezSDK ?? (BreezManager.breezSDK = try BreezManager.connectSDK())
```

Reply:
Doesn't compile for me

Reply:
Right, don't think we can use `try` on right side of a nil coalescing operator... Ignore this.